From e23c2f4260406713fbd95c29d8d2fe8390daf043 Mon Sep 17 00:00:00 2001
From: ling <ling@users.noreply.github.com>
Date: Sat, 22 Aug 2026 11:21:53 +0000
Subject: [PATCH] hide flow offloading type from firewall settings

---
 .../resources/view/firewall/zones.js          | 28 -------------------
 1 file changed, 28 deletions(-)

diff --git a/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js b/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js
index ddd5d5c2f4..8081411688 100644
--- a/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js
+++ b/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js
@@ -73,34 +73,6 @@ return view.extend({
 			popt.value('ACCEPT', _('accept'));
 		}
 
-		/* Netfilter flow offload support */
-
-		if (L.hasSystemFeature('offloading')) {
-			s = m.section(form.TypedSection, 'defaults', _('Routing/NAT Offloading'),
-				_('Not fully compatible with QoS/SQM.'));
-
-			s.anonymous = true;
-			s.addremove = false;
-
-			o = s.option(form.RichListValue, "offloading_type", _("Flow offloading type"));
-			o.value('0', _("None"));
-			o.value('1', _("Software flow offloading"), _('Software based offloading for routing/NAT.'));
-			o.value('2', _("Hardware flow offloading"), _('Hardware based offloading for routing with/without NAT.') + ' ' + _(' Requires hardware NAT support.'));
-			o.optional = false;
-			o.load = function (section_id) {
-				const flow_offloading = uci.get('firewall', section_id, 'flow_offloading');
-				const flow_offloading_hw = uci.get('firewall', section_id, 'flow_offloading_hw');
-				return (flow_offloading === '1')
-					? (flow_offloading_hw === '1' ? '2' : '1')
-					: '0';
-			};
-			o.write = function(section_id, value) {
-				uci.set('firewall', section_id, 'flow_offloading', value === '0' ? null : '1');
-				uci.set('firewall', section_id, 'flow_offloading_hw', value === '2' ? '1' : null);
-			};
-		}
-
-
 		s = m.section(form.GridSection, 'zone', _('Zones'));
 		s.addremove = true;
 		s.anonymous = true;
-- 
2.53.0

